rllib/utils/sgd.py

```diff
@@ -15,6 +15,8 @@
 def averaged(kv):
```

Comment:
Couldn't we do this with tree/tf.nest.map_structure instead?

Reply:
Hmm, it isn't a recursive mapping, so is there an advantage of using tree?